src/main/java/uk/gov/hmcts/reform/jps/domain/JudicialOfficeHolder.java

```diff
@@ -30,4 +38,35 @@ public class JudicialOfficeHolder {
     @Column(name = "personal_code")
     private String personalCode;
 
+    @ToString.Exclude
```

Comment:
Please replace @Data with @Setter and @Getter as this causes problems during fetching records